game/views.py

```
@@ -24,142 +28,167 @@
 from .engine import ChessGame
 from .models import GameResult
 
+logger = logging.getLogger(__name__)
+
+
+def _count_legal_moves(game):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_count_legal_moves`

(ANN202)